add assembly wb

drwho495 opened the pull request:
Add the Ondsel based assembly wb. This is my first time working on an open source project, so I don't really know what I'm doing.

drwho495 committed:
add assembly wb

drwho495 committed:
Add sticky selections to path jobs

realthunder commented:
Hi, looks like you just pulled in Ondsel solver. I happen to be in the middle of merging upstream FreeCAD which included Ondsel code as git submodule, so there will be conflict there. Please wait till I finish the merge first, then we'll see how to proceed.

drwho495 commented:
that is weird, I changed some of the file in gui and added the assembly
folder to src/Mod you can look at the fork I made here,
https://github.com/drwho495/my-FreeCAD-Fork

On Tue, Apr 16, 2024 at 8:43 AM Zheng Lei ***@***.***> wrote:

> Hi, looks like you just pulled in Ondsel solver. I happen to be in the
> middle of merging upstream FreeCAD which included Ondsel code as git
> submodule, so there will be conflict there. Please wait till I finish the
> merge first, then we'll see how to proceed.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/realthunder/FreeCAD/pull/993#issuecomment-2059129243>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQZQJYUGRIAMYQLI65NM2QDY5UTJPAVCNFSM6AAAAABGGQ62N2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANJZGEZDSMRUGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

drwho495 committed:
Update README.md

drwho495 committed:
Update README.md

drwho495 committed:
add assembly context creator

drwho495 committed:
Update README.md

drwho495 committed:
Update README.md

drwho495 committed:
Update README.md

drwho495 committed:
Add Assembly Context Update tool

drwho495 committed:
add ex2

drwho495 committed:
Update README.md

drwho495 committed:
Update README.md

drwho495 committed:
Delete assembly-context-ex2.mp4

drwho495 committed:
add ex2 gif

drwho495 committed:
Update README.md

drwho495 committed:
bugfixes

drwho495 committed:
Merge branch 'LinkStable' of https://github.com/drwho495/FreeCAD-Context-Fork into LinkStable

drwho495 committed:
Add Realthunder's ubuntu build here

drwho495 committed:
add build script

drwho495 committed:
fix missing reference, add experimental mode

drwho495 committed:
Merge branch 'realthunder:LinkStable' into LinkStable
